core/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project(sample-core)
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 2.6)
+
```

Comment:
To avoid a warning with cmake 3.19 we should put `cmake_minimum_required(VERSION 3.0)`